cinema_service/urls.py

```diff
@@ -2,10 +2,20 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (SpectacularAPIView,
+                                   SpectacularSwaggerView,
+                                   SpectacularRedocView)
```

Comment:
let's split these imports by starting them from the next line each one

Reply:
Fixed